docs/core/additional-tools/uninstall-tool.md

```diff
@@ -0,0 +1,357 @@
+ms.date: 11/12/2019
+---
+# .NET Core Uninstall Tool
+
+The [.NET Core Uninstall Tool](https://github.com/dotnet/cli-lab/releases) (`dotnet-core-uninstall`) lets you clean up .NET Core SDKs and runtimes on a system such that only the desired versions remain. A collection of options is available to specify which versions you want to uninstall.
+
+The tool supports Windows and macOS. Linux is currently not supported.
+
+Note that the Windows version of this tool can only uninstall .NET Core SDKs and runtimes that were installed using .NET Core SDK and runtime installers. Prior to Visual Studio 2019 Update 16.3, the Visual Studio installer called the .NET Core SDK installer and these versions can be uninstalled with the tool. The `dotnet-core-uninstall list` command displays which SDKs can be uninstalled with the tool.
+
+## Install the tool
+
+You can download the .NET Core Uninstall Tool from [GitHub Releases](https://github.com/dotnet/cli-lab/releases).
+
+> [!NOTE]
+> The tool requires elevation to uninstall .NET Core SDKs and runtimes. Therefore, it should be installed in a write-protected directory such as *C:\Program Files* on Windows or */usr/local/bin* on macOS. See also [Elevated access for dotnet commands](../tools/elevated-access.md). Detailed installation instructions are available on the [GitHub Releases page](https://github.com/dotnet/cli-lab/releases).
+
+## How to use the tool
+
+1. Display installed .NET Core SDKs and runtimes that can be removed by this tool using `dotnet-core-uninstall list`.
+2. Use `dotnet-core-uninstall dry-run` to ensure the right SDKs and runtimes will be uninstalled.
+3. Use `dotnet-core-uninstall remove` to uninstall .NET Core SDKs and runtimes. This command must be run with Admin privileges.
+4. Delete the `NuGetFallbackFolder` if you no longer need it (optional).
+
+### Display installed .NET Core SDKs and runtimes
+
+Use the `dotnet-core-uninstall list` command lists installed .NET Core SDKs and runtimes that can be removed with this tool. Some SDKs and runtimes may be required by Visual Studio and they're displayed with a note of why it isn't recommended to uninstall them.
+
+On Windows, the tool can only uninstall SDKs and runtimes that were installed using the .NET Core SDK and runtime installer, or the Visual Studio installer prior to Visual Studio 2019 16.3. On macOS, the tool can only uninstall SDKs and runtimes located in the */usr/local/share/dotnet* folder. This tool doesn't work on Linux. Because of these limitations, the tool may not be able to uninstall all of the .NET Core SDKs and runtimes on your machine. You can use the `dotnet --info` command to find all of the .NET Core SDKs and runtimes available, including those SDKs and runtimes that this tool can't remove.
+
+#### Synopsis
+
+```console
+dotnet-core-uninstall list [options]
+```
+
+#### Options
+
+## [Windows](#tab/windows)
+
+* **`--aspnet-runtime`**
+
+  Lists all the ASP.NET Core runtimes that are installed on your machine.
```

Comment:
The tool does not necessarily list all runtimes/sdks installed on the machine- for example it doesn't list bundles installed via zips. More accurate wording here would be that it lists all runtimes that are able to be uninstalled by this tool.

Reply:
Ok, makes sense.